README.md

```diff
@@ -78,7 +78,7 @@ For this portfolio template design I am using mobile first approach.
 
 > You can live demo form here.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link]https://imranhossainemi.github.io/portfolio/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] [OPTIONAL] Please update your Readme.md file as we need to make sure we document our work professionally, currently you do not have a License section and an MIT or any othr kind of license assosciated with your project.